sklearn/cluster/_optics.py

```diff
@@ -233,6 +232,8 @@ class OPTICS(ClusterMixin, BaseEstimator):
     >>> clustering = OPTICS(min_samples=2).fit(X)
     >>> clustering.labels_
     array([0, 0, 0, 1, 1, 1])
+
+    For an example of usage, see :ref:`sphx_glr_auto_examples_cluster_plot_optics.py`.
```

Comment:
```suggestion

    For an example of usage, see
    :ref:`sphx_glr_auto_examples_cluster_plot_optics.py`.
```

Reply:
@greyisbetter Thanks, I tried to follow all of the suggestion but still somehow it couldn't pass all the checks!